examples/v1beta1/100-cpu-limit.yaml

```diff
@@ -0,0 +1,28 @@
+# This example NodePool limits the amount of compute
+# NodePool by Karpenter to 100 CPU cores
+apiVersion: karpenter.sh/v1beta1
+kind: NodePool
+metadata:
+  name: default
+spec:
+  spec:
+    template:
+      spec:
+        nodeClassRef:
+          name: default
+  limits:
+    cpu: 100
+---
+apiVersion: karpenter.k8s.aws/v1beta1
+kind: EC2NodeClass
+metadata:
+  name: default
```

Comment:
This could change to:
```yaml
  name: amazonlinux2
  annotations:
    kubernetes.io/description: "Nodes running Amazon Linux 2"
```

Reply:
I like adding the description annotations to each of these examples @jonathan-innis

Reply:
Done!